styles/mixins/patterns/_molecules.scss

```diff
@@ -59,3 +59,32 @@ $-space: var.$layout-space;
     @include layout.vertical-rhythm($-space * 0.5);
   }
 }
+
+// A responsive sized and positioned container for modal content
+@mixin modal {
+  // The modal container is positioned horizontally- and vertically-centered
+  @include layout.fixed-centered;
+
+  // This sizing applies to smaller viewports
+  width: 90vw;
+  max-width: 90vw;
```

Comment:
I noticed a jump as the dialog shrinks in size when going from a width that is less than the `min-width` media query below to one that is greater.

Reply:
What would your preferred behavior be? I'm never sure how best to optimize for the "but it isn't animated/smooth when resizing a screen" issue...